setup-env.sh

```diff
@@ -0,0 +1,3 @@
+BASEDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+export PATH=${BASEDIR}/build/node/bin:${BASEDIR}/node_modules/.bin:${PATH}
```

Comment:
Just copying this from the `packs` repo, allows us to run `ts-node` unqualified when trying to test the CLI locally. (In the `packs` repo it allows us to run `coda` unqualified.)

Reply:
Maybe put that as a comment in the code for future reference?

Reply:
Done